### ISSUE TITLE
Add parameters to set the default values for the layout/panels options #60

### ISSUE THREAD

**Issue by AndreJorgeLopes** [mlange-42/git-igitt#56]
I would love to have the possibility to open the branch list and use the vertical layout without needing to press B and L.

**Comment by mlange-42** [comment]
Generally, a good idea.

As an alternative, how about saving the layout on a per-repo basis, as it is currently done for the branching model?
That seems more convenient than calling with parameters.

**Comment by AndreJorgeLopes** [comment]
Is that already implemented ? If so, could you give me a little guide ? Because I tried to search a bit about it, but I didn't found anything about that on the project documentation.
Otherwise, if the functionality is still going to by built, I would opine that settings regarding styling, would be better put on the parameters, since the "better" styling is highly subjective topic, even within the same project.

**Comment by mlange-42** [comment]
@AndreJorgeLopes No, it is not implemented. Is was meant as a possible alternative option.
